spec/abstract-jobs.html

```diff
@@ -102,6 +102,7 @@ <h1> FinalizationGroupCleanupJob ( _finalizationGroup_ [ , _callback_ ] ) </h1>
       1. Perform ? Call(_callback_, *undefined*, _iterator_).
       1. Return *undefined*.
     </emu-alg>
+    <emu-note type=editor>If calling the callback throws an error, this will be caught within the RunJobs algorithm and reported to the host. HTML does not apply the RunJobs algorithm, but will also report the error.</emu-note>
```

Comment:
Isn’t html required to have the same observable effect as RunJobs, whether it follows those steps exactly or not?

Reply:
That's only the case when this job is invoked from `DoAgentFinalization`. If invoked by `FinalizationGroup.prototype.cleanupSome`, the error will go up to user code normally.

Reply:
@ljharb There's a standing disagreement about the layering. I hope we can revisit it in a future TC39 meeting, but I'm waiting for some ongoing HTML work to complete before doing that. This is just a note, and I'm doing the best I can in the face of this disagreement.

@mhofman Good point, I've clarified this.

Reply:
@littledan I’m aware of that, but that doesn’t mean 262 makes/should mak is mention of it.

Reply:
@ljharb I hope to resolve the layering issue before Stage 4; we can remove the reference if it's not fixed in time, but I think the clarity is important.